testing/python/fixture.py

```diff
@@ -417,6 +417,7 @@ def test_lookup_error(unknown):
             "*1 error*",
         ])
         assert "INTERNAL" not in result.stdout.str()
+        assert 'monkeypatch:session' not in result.stdout.str()
```

Comment:
Talking about performance.... 😹 

We could probably retrieve ``stdout`` just once here:

```python
stdout_text = result.stdout.str()
assert "INTERNAL" not in stdout_text
assert 'monkeypatch:session' not in stdout_text
```